tests/common/dnspolicy/dnspolicy_controller_multi_cluster_test.go

```diff
@@ -1,714 +0,0 @@
-//go:build integration
-
-package dnspolicy
```

Comment:
why did we end up removing this as part of this work?

Reply:
These tests were specifically for the multi cluster status on the gateway (ACM). Since we are removing that support I'd rather just remove the tests than update them and then remove them later. 

Note this change is also partially dropping support for multicluster since there will be no longer any way to set geos/weights for individual clusters via labels on the ACM gateway so a lot of the tests are redundant now anyway.